src/lib/inspector.js

```diff
@@ -97,22 +97,14 @@ Inspector.prototype = {
 
     this.open();
   },
-/*
-  removeObject: function (object) {
-    if (object.parent === null) return; // avoid deleting the camera or scene
-
-    var scope = this;
 
-    object.traverse(function (child) {
-      scope.removeHelper(child);
+  removeObject: function (object) {
+    // Remove just the helper as the object will be deleted by Aframe
+    object.traverse(child => {
```

Comment:
Yep, I was refactoring that function and didn't realised that it was just the function call hehe. 
Actually I believe it should be: `object.traverse(this.removeHelper).bind(this)`